### PR TITLE
Android issue139 bridge activity

### PR DESCRIPTION
**Pull request by haozturk**
As denoted in [issue139](https://github.com/bounswe/bounswe2019group5/issues/139), I have created a bridge activity on which user can land after the a successful login and register and choose a language for proficiency exam.

### CODE REVIEW

File: languageLearning/app/src/main/java/com/example/languageLearning/BridgeActivity.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import android.os.Bundle;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class BridgeActivity extends AppCompatActivity {
+
+    Intent i = new Intent(this, ProfExamActivity.class);
```
Comment (muzunpostalci) [inline languageLearning/app/src/main/java/com/example/languageLearning/BridgeActivity.java]: When I try to navigate to this activity, app stops working and shows this line as error.
When I tried putting this line separately to all functions it worked.

Comment (haozturk) [reply languageLearning/app/src/main/java/com/example/languageLearning/BridgeActivity.java]: Thanks for pointing out the problem. I solved the issue as you suggested. Does it work now?

Comment (muzunpostalci) [reply languageLearning/app/src/main/java/com/example/languageLearning/BridgeActivity.java]: Now its working without errors. Thanks

**Comment by muzunpostalci** [comment]
It is working as intended. I am merging

**Pull request merged**